src/components/templates/Camp/components/RoomModalOngoingEvent.tsx

```diff
@@ -42,18 +42,49 @@ export const RoomModalOngoingEvent: React.FunctionComponent<PropsType> = ({
             </div>
           </div>
           <div className="event-description">{eventToDisplay.description}</div>
-          <a
-            className="btn btn-primary room-entry-button"
-            onClick={() => enterRoom()}
-            id={`enter-room-in-ongoing-event-card-${room.title}`}
-            href={room.url}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Join the room
-          </a>
         </>
       )}
+      {eventToDisplay && !currentEvent && (
+        <>
+          <div className="title-container">
+            <img
+              src="/sparkle-icon.png"
+              className="sparkle-icon"
+              alt="sparkle-icon"
+            />
+            What's on next
```

Comment:
Can you please clean up the `whatsOnText` while in here?

More broadly let's keep an eye out for refactors leaving things in a half finished state. Cheers!